### PR TITLE
Electron-391 (Redesigned Windows & macOS installer)

### PR DESCRIPTION
**Pull request by KiranNiranjan**
## Description
Redesigned Windows installer [ELECTRON-391](https://perzoinc.atlassian.net/browse/ELECTRON-391)

### (**Windows**)
electron-391_1
electron-391_2
electron-391_3
electron-391_4

### (**Mac**)
screen shot 2018-04-30 at 3 47 35 pm
screen shot 2018-04-30 at 3 47 49 pm
screen shot 2018-04-30 at 3 48 02 pm

## Spectron test results
[Electron-391 — Spectron.pdf](https://github.com/symphonyoss/SymphonyElectron/files/1960168/Electron-391.Spectron.pdf)

## Unit test results
[Electron-391 — Unit Tests.pdf](https://github.com/symphonyoss/SymphonyElectron/files/1960201/Electron-391.Unit.Tests.pdf)

## Open Questions if any and Todos
- [X] Unit-Tests (No Changes)
- [X] Documentation
- [X] Automation-Tests
When solved, check the box and explain the answer.

@VishwasShashidhar @VikasShashidhar Please review. Thanks 😃

### CODE REVIEW

File: installer/mac/intro.html [hunk 1]
```diff
@@ -1,11 +1,18 @@
...
 <html>
 <head>
-	<title>Welcome to the Symphony Installer</title>
+	<title>Thanks for downloading Symphony</title>
 </head>
 <body>
 <br/>
 <br/>
-<p style="font-size: 13px; font-family: Sans-Serif; color: #000">&nbsp;&nbsp;&nbsp;We'll walk you through the steps to install Symphony</p>
+<p style="font-size: 13px; font-family: Sans-Serif; color: #000">
```
Comment (VikasShashidhar) [inline installer/mac/intro.html]: I'd like for this to use the same fonts/fallbacks that the web app is using. Can you please take a look?

File: installer/mac/intro.html [hunk 2]
```diff
@@ -1,11 +1,18 @@
...
 <br/>
 <br/>
-<p style="font-size: 13px; font-family: Sans-Serif; color: #000">&nbsp;&nbsp;&nbsp;We'll walk you through the steps to install Symphony</p>
+<p style="font-size: 13px; font-family: Sans-Serif; color: #000">	
+	&nbsp;&nbsp;&nbsp;Symphony allows you to:<br>
+	<ul>
+		<li>Collaborate securely</li>
+		<li>Communicate via messages, voice, and video</li>
+		<li>Connect with colleagues and coworkers</li>
+	</ul>
```
Comment (VikasShashidhar) [inline installer/mac/intro.html]: Has no design been provided for this? I mean in terms of spacing, etc.?

File: package.json [hunk 3]
```diff
@@ -88,7 +88,7 @@
     "browserify": "14.5.0",
     "cross-env": "3.2.4",
     "electron": "1.8.4",
-    "electron-builder": "13.11.1",
+    "electron-builder": "20.10.0",
```
Comment (VikasShashidhar) [inline package.json]: Need to ensure that we don't run into any regressions with this change.

File: installer/mac/intro.html (hunk 1 above)
Comment (VishwasShashidhar) [reply installer/mac/intro.html]: Installer doesn't support custom fonts. Only the ones available by default on the system.

File: installer/mac/intro.html (hunk 2 above)
Comment (VishwasShashidhar) [reply installer/mac/intro.html]: No, not for the mac installer.

File: package.json (hunk 3 above)
Comment (VishwasShashidhar) [reply package.json]: Already tested, no regressions. Created builds on Windows.

File: installer/mac/intro.html (hunk 2 above)
Comment (VikasShashidhar) [reply installer/mac/intro.html]: Can we ask for one?

File: installer/mac/intro.html (hunk 1 above)
Comment (VikasShashidhar) [reply installer/mac/intro.html]: Yes. Please refer to https://css-tricks.com/snippets/css/system-font-stack/

File: installer/mac/intro.html (hunk 2 above)
Comment (VishwasShashidhar) [reply installer/mac/intro.html]: Spoke to the design team and they suggested that as long as the installer works with closest design to Windows, it should be fine.

**Pull request merged**